apps/precursor-chemicals/translations/src/en/pages.json

```diff
@@ -74,7 +74,10 @@
   },
 
   "upload-conduct-certificate": {
-    "header": "Upload a certificate of good conduct"
+    "header": "Upload a certificate of good conduct",
+    "intro": "Upload a scanned copy of your certificate of good conduct or other personal reference.",
+    "no-files-uploaded": "No files uploaded",
+    "files-uploaded": "Files uploaded"
```

Comment:
I'm wondering seeing that these two texts are the same for all file upload components whether we should move these two values into `fields.file-upload`. This would mean updating the view template to accept the change. What do you think?

Reply:
...and pulling the values from the equivalent `pages.upload-company-certificate` and updating its own template too :)

Reply:
Thanks for this suggestion Jamie. I will make the change.

Reply:
Changes look good